### ISSUE TITLE
The email commands don't work with pnpm, including being able to preview emails

### ISSUE THREAD

**Issue by matt-aitken** [zenorocha/react-email#317]
# Reproduction steps
1. Create a brand new project using `npx create-email@latest`
2. Run `pnpm install` in the root
3. Run `pnpm run dev` 

It fails with this error:
```sh
matt@Matts-16-MBP react-email-starter-pnpm % pnpm run dev

> emails@0.0.0 dev /Users/matt/Desktop/email test/react-email-starter-pnpm
> email dev

 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by matt-aitken** [comment]
I think I've found the issue, which is possibly impacting non `pnpm` users as well.

This command is asynchronous and isn't being awaited:
https://github.com/zenorocha/react-email/blob/fc3122a2c58897a11d2da21e3673637a032ec21e/packages/react-email/source/commands/dev.ts#L114

**Comment by matt-aitken** [comment]
Will be fixed by #316
